financial-services/1.8/modules/ROOT/pages/release-notes.adoc

```diff
@@ -1,9 +1,21 @@
 = Release Notes
 :fins-version: {page-component-version}
 
-== v1.8
+== 1.9
```

Comment:
If you are using the same system for versioning as we use for connectors, you will need to create the new folder for 1.9. Here, the path is still showing 1.8. Maybe you do things differently on the Accelerators side, but if you do, should you update so this path shows 1.9?